src/main/host/thread.rs

```diff
@@ -302,9 +310,9 @@ impl Thread {
         &self,
         ctx: &ProcessContext,
         flags: libc::c_ulong,
-        child_stack: ForeignPtr,
-        ptid: ForeignPtr,
-        ctid: ForeignPtr,
+        child_stack: ForeignPtr<u8>,
```

Comment:
I think `<()>` might make more sense here? I think that makes it clearer that it's an opaque pointer of unknown size, not a pointer to a single `u8`.

Reply:
Good idea, added.